mne/realtime/lsl_client.py

```diff
@@ -83,8 +83,18 @@ def iter_raw_buffers(self):
             yield np.vstack(samples).T
 
     def _connect(self):
+<<<<<<< HEAD
+<<<<<<< HEAD
         # To use this function with an LSL stream which has a 'name' but no
         # 'source_id', change the keyword in pylsl.resolve_byprop accordingly.
+=======
+        """To use this function with an LSL stream which has a 'name' but no 
+        'source_id', change the keyword in pylsl.resolve_byprop accordingly."""
+>>>>>>> Comment use of keywords in pylsl.resolve_byprop.
+=======
+        # To use this function with an LSL stream which has a 'name' but no
+        # 'source_id', change the keyword in pylsl.resolve_byprop accordingly.
+>>>>>>> STY: Comments to please the pedantic CIs
```

Comment:
@akovrig you need to fix your merge conflicts. Maybe start from a clean master and reapply your fix?
let us know if you need help. thanks

Reply:
Here you go: https://github.com/mne-tools/mne-python/pull/6217